wsd/COOLWSD.cpp

```diff
@@ -2802,15 +2800,24 @@ void COOLWSD::innerInitialize(Application& self)
     {
         conf.setString("feedback.show", "true");
         conf.setString("welcome.enable", "true");
-        COOLWSD::MaxConnections = MAX_CONNECTIONS;
+        COOLWSD::MaxConnections = net::Config::get().MaxConnections; // aligned w/ MAX_CONNECTIONS above
```

Comment:
Reason for now net::Config was to have things configurable at least for the unit test, proving functionality.
Then - after your hint - I found your MAX_CONNECTION and realized that it must be considered and used, if not overridden by the optional config layer (unit test).
Hence, if we still want to add this functionality of added cut-off after max-connections - we should test it.
(Now I saw a unit test building up MAX_CONNECTION .. though, so it might indeed not be required)

Considering your MAX_CONNECTION measures and if that is sufficient, this angle of limited connection in this patch series probably shall be removed completely?

Reply:
revised version has no more xml config file side-effects, only storing MAX_CONNECTIONS in the net::Defaults - potentially mutated by unit test only.